packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -119,11 +123,22 @@ const pointers = defineQuery([InputPointerComponent, InputSourceComponent, Not(X
 const xrSpaces = defineQuery([XRSpaceComponent, TransformComponent])
 const spatialInputSourceQuery = defineQuery([InputSourceComponent, TransformComponent])
 const inputSourceQuery = defineQuery([InputSourceComponent])
+const nonSpatialInputSourceQuery = defineQuery([InputSourceComponent, Not(TransformComponent)])
 const inputs = defineQuery([InputComponent])
 
+const worldPosInputSourceComponent = new Vector3()
+const worldPosInputComponent = new Vector3()
+
 const inputXRUIs = defineQuery([InputComponent, VisibleComponent, XRUIComponent])
 const inputBoundingBoxes = defineQuery([InputComponent, VisibleComponent, BoundingBoxComponent])
 const inputObjects = defineQuery([InputComponent, VisibleComponent, GroupComponent])
+const spatialInputObjects = defineQuery([
+  InputComponent,
+  VisibleComponent,
+  TransformComponent,
+  Not(CameraComponent),
+  Not(AvatarComponent)
```

Comment:
we need to find a different way of handling this, avatars must not be included in the spatial package

Reply:
removed references to AvatarComponent